### PR TITLE
Label constructor now takes angle as an optional parameter.

### LINKED ISSUE #2130: Label constructor should take angle
Missed this when #2092 went in, but `Label`s should take an angle in their constructor, the way they used to take an `orientation` in the constructor.

### PR DESCRIPTION
**Pull request by jtlan**
..the same way that the old constructor used to take the orientation as a parameter.

Close #2130.

### CODE REVIEW

**Comment by crmorford** [comment]
Reason for this? 
If i see a number with the parameter name "angle" as a parameter, I'm going to expect to be able to put in anything 0-360. The strings felt discrete, this feels like it should be continuous and I find it a bit misleading.

Also confused about why don't 180 and 270 work? I tried 270 before -90, it felt more intuitive to me.

**Comment by jtlan** [comment]
Originally the API was changed (in #2092) because of #2048. If later we want to support labels at tilted angles (45 degrees, etc), passing in the angle seems more expandable. It's also closer in line with the `tickLabelAngle()` API point on `Axes.Category`.

**Comment by crmorford** [comment]
Hm okay I see the advantages if we want to support tilted angles later. 
But can we also handle 270 correctly? That's more natural to me.

**Comment by jtlan** [comment]
>But can we also handle 270 correctly? That's more natural to me.
Yeah, we can add a parser that converts arbitrarily large numbers to a valid range.

File: src/components/label.ts [hunk 1]
```diff
@@ -19,12 +19,13 @@ export module Components {
      *
      * @constructor
      * @param {string} displayText The text of the Label (default = "").
+     * @param {number} angle The rotation angle of the text (default = 0).
```
Comment (aicioara) [inline src/components/label.ts]: Mention that 0 means horizontal. As a reader of this documentation, I have no idea what a rotation of 0 means in this library

Comment (aicioara) [inline src/components/label.ts]: (I hope it means horizontal)

**Comment by aicioara** [comment]
Rest looks good

**Pull request merged**